src/V3Class.cpp

```diff
@@ -78,10 +82,12 @@ class ClassVisitor : public AstNVisitor {
         VL_RESTORER(m_prefix);
         {
             m_classScopep = classScopep;
+            m_packageScopep = scopep;
             m_prefix = nodep->name() + "__02e";  // .
             iterateChildren(nodep);
         }
         m_classScopep = nullptr;
+        m_packageScopep = nullptr;
```

Comment:
Remove these and put a VL_RESTORER for them above (under m_prefix).  m_classScopep should have already been that way, missed it at some point.  (Didn't break anything as we don't support classes declared under classes yet.)